core/jazz_cloud-logs-streamer/package.json

```diff
@@ -6,13 +6,16 @@
     "private": true,
     "author": [],
     "scripts": {
+        "start": "node index",
         "test": "mocha"
     },
     "dependencies": {
-        "unicode-byte-truncate": "*"
+        "node-stream-zip": "^1.8.2",
```

Comment:
Shouldn’t this be devdependency?

Reply:
I don't see any change but the comment was resolved.

Reply:
removing this line results in not being able to start the node package manager

Reply:
The comment is about line 12+

Reply:
removed the '"node-stream-zip": "^1.8.2"," from the dependency surya

Reply:
is that what you were looking for? i was not entirely sure, please let me know